doc/source/reference/routines.char.rst

```diff
@@ -3,9 +3,9 @@ String operations
 
 .. currentmodule:: numpy.core.defchararray
 
-This module provides a set of vectorized string operations for arrays
-of type `numpy.string_` or `numpy.unicode_`.   All of them are based on
-the string methods in the Python standard library.
+The `numpy.core.defchararray` module provides a set of vectorized string
+operations for arrays of type `numpy.string_` or `numpy.unicode_`.
+All of them are based on the string methods in the Python standard library.
```

Comment:
The module name should be `numpy.char` which is a alias to `numpy.core.defchararray`. Also on line 4. Please make sure the links work (italicized text instead of a link is a giveaway), following the workflow at [how to build documentation](http://www.numpy.org/devdocs/docs/howto_build_docs.html)

Reply:
Thanks for review @mattip ! I have changed the module name and when we click on it, it points to the same page (since it on the numpy.core.defchararray module page only). Let me know if this is what you expected.

Reply:
Does the proper links get created if you do this?
```
.. currentmodule:: numpy.char

.. module:: numpy.char

The `numpy.char` module provides  ...
```

Reply:
Yes, those changes also worked. I have done this change in new commit. It is reflected to all string operation methods as well.

e.g. 

`numpy.core.defchararray.multiply` will be now `numpy.char.multiply` here : https://www.numpy.org/devdocs/reference/generated/numpy.core.defchararray.multiply.html#numpy.core.defchararray.multiply